### PR TITLE
add script to provision machine and run tests

### PR DESCRIPTION
**Pull request by richm**

### CODE REVIEW

**Comment by nhosoi** [comment]
/lgtm

Thanks, @richm !!

**Comment by richm** [comment]
@jcantrill I think this should be merged anyway, if folks think it is useful, even if the tests fail.

File: hack/get-machine-run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,580 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+usage() {
+    local bn=$( basename $0 )
+    echo Usage: $0
+    echo Assumes you have $HOME/origin-aggregated-logging and $HOME/openshift-ansible
```
Comment (jcantrill) [inline hack/get-machine-run-tests.sh]: Can we not modify this to be a var other then $HOME.  I, for example, use $HOME/git

File: hack/get-machine-run-tests.sh [hunk 2]
```diff
@@ -0,0 +1,580 @@
+#!/bin/bash
+
```
Comment (jcantrill) [inline hack/get-machine-run-tests.sh]: Can we add a comment here describing the purpose.

**Pull request merged**